pymodbus/repl/client/main.py

```diff
@@ -178,7 +178,7 @@ def _process_args(args, string=True):
                             complete_while_typing=True,
                             bottom_toolbar=bottom_toolbar,
                             key_bindings=use_keys,
-                            history=FileHistory('../.pymodhis'),
```

Comment:
I was planning to use a directory in users `HOME` directory and save the history file there. So that it would be available no matter from where the REPL is invoked.